nextflow.config

```
@@ -252,8 +252,9 @@ params {
     // Genotyping
     run_genotyping                           = false
     genotyping_tool                          = null
-    genotyping_source                        = null
-    skip_bcftools_stats                      = false
+    genotyping_source                        = 'raw'
```

```suggestion
    genotyping_source                        = null
```
Setting this to `null` was done on purpose, because users would often activate bam trimming or rescaling and not realise their genotypes were still from the raw bams. This way, they get an error if they try genotyping without explicitly setting the source.